scala-package/core/src/main/scala/org/apache/mxnet/io/MXDataIter.scala

```diff
@@ -151,12 +175,34 @@ private[mxnet] class MXDataIter(private[mxnet] val handle: DataIterHandle,
     out.value
   }
 
+  /**
+    * Get the DType
+    * @return DType
+    */
+  def getDType(): DType = {
```

Comment:
suggest to add two new parameters to the constructor `dataDescriptor` and `labelDescriptor`.
Make sure not break API backwards compatibility